subxt/src/config.rs

```diff
@@ -97,4 +105,46 @@ impl Config for DefaultConfig {
         sp_runtime::generic::Header<Self::BlockNumber, sp_runtime::traits::BlakeTwo256>;
     type Signature = sp_runtime::MultiSignature;
     type Extrinsic = sp_runtime::OpaqueExtrinsic;
+    type ExtrinsicParams = crate::tx::SubstrateExtrinsicParams<Self>;
+}
+
+/// Default set of commonly used types by Polkadot nodes.
+pub type PolkadotConfig = WithExtrinsicParams<
+    SubstrateConfig,
```

Comment:
I don't know how that works but for polkadot the address type is different https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L1488

Reply:
Ooohhh is it? 

It looks like both are `MultiAddress<AccountId32, _>`'s at least, but that second generic looks like a u32 in substrate and () in polkadot? I guess this means that the byte encoding is identical but polkadot doesn't support Index addresses.

Maybe we should change the PolkadotConfig to match this then, just to make sure people use the right types?

Reply:
yeah, I meant that that second generic param is different as you said.

doesn't seem to have any implications when I have used subxt but might be better match the one in polkadot ^^

Reply:
Yeah; I guess it has no impact on the SCALE encoding, so you'd only bump into issues trying to pass eg a `MultiAddress::Index(123)` in to a polkadot node. Otherwise you'll be fine.

I'll add a separate issue to fix it; it's been like this for ages :)